src/Provider/GitLab.php

```diff
@@ -0,0 +1,60 @@
+<?php
+/**
+ * @see       https://github.com/phly/keep-a-changelog for the canonical source repository
+ * @copyright Copyright (c) 2018 Matthew Weier O'Phinney
+ * @license   https://github.com/phly/keep-a-changelog/blob/master/LICENSE.md New BSD License
+ */
+
+declare(strict_types=1);
+
+namespace Phly\KeepAChangelog\Provider;
+
+use Gitlab\Client as GitLabClient;
+
+class GitLab implements ProviderInterface
+{
+    const PACKAGE_NAME = 'm4tthumphrey/php-gitlab-api';
+
+    /**
+     * @inheritDoc
+     */
```

Comment:
Considering that the docblocks for the interface only duplicate the information in the typehints, we can remove those, and then remove these `@inheritDoc` entries. Alternately, give _descriptions_ to the docblocks in the interface.